src/middlewares/userValidations.js

```diff
@@ -31,8 +31,7 @@ const validateUser = path => (req, res, next) => {
  * @returns {object} JSON response
  */
 const emailExists = (req, res, next) => {
-  let { email } = req.body;
-  email = email.trim().toLowerCase();
+  const { email } = req.body;
```

Comment:
`email` has been trimmed and changed to lowercase.

Reply:
Changes implemented